File: resources/templates/javascript/stub/http_handler_response_received.js
var RequestToBeSentAction = Packages.burp.api.montoya.http.handler.RequestToBeSentAction;
var ResponseReceivedAction = Packages.burp.api.montoya.http.handler.ResponseReceivedAction;
var HttpRequestToBeSent = Packages.burp.api.montoya.http.handler.HttpRequestToBeSent;
var HttpResponseReceived = Packages.burp.api.montoya.http.handler.HttpResponseReceived;

function handleHttpResponseReceived(montoyaApi, httpResponseReceived) {
    return ResponseReceivedAction.continueWith(httpResponseReceived.withAddedHeader("DemoHttpHandler","Added by user defined script (js)"));
}

File: resources/templates/jython/stub/http_handler_response_received.py
from burp.api.montoya.http.handler import RequestToBeSentAction
from burp.api.montoya.http.handler import ResponseReceivedAction
from burp.api.montoya.http.handler import HttpRequestToBeSent
from burp.api.montoya.http.handler import HttpResponseReceived

def handleHttpResponseReceived(montoyaApi, httpResponseReceived):
    return ResponseReceivedAction.continueWith(httpResponseReceived.withAddedHeader("DemoHttpHandler","Added by user defined script (py)"))